tests/smoke/events.js

```diff
@@ -93,7 +95,7 @@
 				var secondParagraph = editor.editable().getChild(1);
 
 				// no ajax call required on the second run, since words are repeats.
-				observer.assert(["spellCheckComplete", "startMarkTypos", "startScanWords"]);
+				observer.assert(["spellCheckComplete", "startRender", "startScanWords"]);
 				observer.assertRootIs(secondParagraph);
 			}
```

Comment:
For our own sanity, this might be a good time to have a test helper that clears `suggestions` and `suggestionscache`, which we could run in the teardown message.  Then we don't need to keep creating new variations of 'foo' and 'asdf' for new tests.